flo_ai/router/flo_linear.py

```diff
@@ -61,10 +61,11 @@ def create(session: FloSession, name: str, team: FloTeam):
         return FloLinear.Builder(session=session, name=name, flo_team=team).build()
 
     class Builder:
-        def __init__(self, session: FloSession, name: str, flo_team: FloTeam) -> None:
+        def __init__(self, session: FloSession, name: str, flo_team: FloTeam, model_nick_name: str = 'default') -> None:
             self.name = name
             self.session = session
             self.team = flo_team
+            self.model_name = model_nick_name
 
         def build(self):
-            return FloLinear(self.session, self.name, self.team)
+            return FloLinear(self.session, self.name, self.team,model_name=self.model_name)
```

Comment:
1. add a space after comma
2. The class `FloLinear` needs to accept model_name, you have to add model_name to the class above

Reply:
Remove FloLinear changes

Reply:
resolved